portfolio/src/main/webapp/task.js

```diff
@@ -6,20 +6,130 @@
  * converted into a JSON string.
  */
 class Task {
-  // TODO(raulcruise): could you add the type for each param please? 
-  
+  // TODO(raulcruise): could you add the type for each param please?
+
   /**
    * All fields are required except description, which will be null
    * when not set.
    * @param name: name of task as String
    * @param description: description of task as String, can be null
-   * @param duration: duration of task as Duration class
-   * @param priority: priority level of task, range 1-5 inclusive. 
+   * @param duration: duration of task in minutes as an integer
+   * @param taskPriority: priority level of task, range 1-5 inclusive.
    */
-  constructor(name, description, duration, priority) {
+  constructor(name, description, duration, taskPriority) {
     this.name = name;
     this.description = description;
     this.duration = duration;
-    this.priority = priority;
+    this.taskPriority = taskPriority;
   }
 }
+
+/**
+ * This method grabs the required inputs from the front end of the page that
+ * will populate the Task's information. It requires that the front end fields
+ * for name, and duration to be passed in. Once it checks for the required
+ * inputs, a JavaScript task class is created and passed into the
+ * updateTaskList function.
+ */
+function createNewTask() {
+  const name = document.getElementById('new-task-name').value;
```

Comment:
Consider using jquery here.

Like,
const name = $('#new-task-name').val();
...

I think either val() and text() should work.

Reply:
Done.